handlers/pulp_rpm/handlers/rpmtools.py

```diff
@@ -140,7 +159,11 @@ def install(self, names):
                 yb.processTransaction()
             else:
                 yb.progress.set_status(True)
-            return Package.installed(yb.tsInfo)
+            details = Package.installed(yb.tsInfo)
+            affected = Package.affected(details)
+            map(log.info, [INSTALLED % dict(p=p) for p in affected])
+            map(yb.logfile.info, [INSTALLED % dict(p=p) for p in affected])
+            return details
```

Comment:
I think instead of writing same code for 5 times maybe we could do something like:
```
def affected_packages(action, yb):
        details = eval("Package.%s(yb.tsInfo)" % action)
        affected = Package.affected(details)
        map(log.info, [eval(action.upper()) % dict(p=p) for p in affected])
        map(yb.logfile.info, [eval(action.upper()) % dict(p=p) for p in affected])
        return details
```
and call it like:
```
return Package.affected_packages('installed', yb)
```
How do you think?